source/NVDAObjects/window/winword.py

```diff
@@ -966,8 +966,8 @@ def _normalizeControlField(self, field):
 		return newField
 
 	def _normalizeFormatField(self, field, extraDetail=False):
-		_startOffset = int(field.pop("_startOffset"))
-		_endOffset = int(field.pop("_endOffset"))
+		field.pop("_startOffset")
+		field.pop("_endOffset")
```

Comment:
should these just be removed entirely?
These seem unused in NVDA.

This line should probably be removed too

https://github.com/nvaccess/nvda/blob/8fb8ffcaba3950b1e4ca7b58e165267fc5369eb9/nvdaHelper/remote/winword.cpp#L1293

Thoughts @michaelDCurran ?

Reply:
Yes, those attributes can be removed from the xml stream. They used to be used to recreate an MS Word range object to fetch extra track change revision info such as author and date in _normalizeFormatField, but we no longer report that info here.

Reply:
The code that used them was removed in ca50d9b004 sometime after 2015.4.

Reply:
```suggestion
```

Can you also remove the line in nvdaHelper?

Reply:
> Can you also remove the line in nvdaHelper?

Actually, I have not removed all the line but rather only `_startOffset` and `_endOffset` value in the xml stream.